Source/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -944,6 +944,18 @@ - (void)setInsetsLayoutMarginsFromSafeArea:(BOOL)insetsLayoutMarginsFromSafeArea
   }
 }
 
+- (NSDictionary<NSString *,id<CAAction>> *)actions
+{
+  _bridge_prologue_read;
+  return _getFromLayer(actions);
+}
+
+- (void)setActions:(NSDictionary<NSString *,id<CAAction>> *)actions
+{
+  _bridge_prologue_write;
+  _setToLayer(actions, [actions copy]);
```

Comment:
I don't think you need to copy here as CALayer's actions is: `@property(copy) NSDictionary<NSString *,id<CAAction>> *actions;`

Reply:
True, done.